src/lithium/reducer.py

```diff
@@ -682,85 +682,95 @@ def tryRemovingChunks(self, chunkSize, testcase, interesting, tempFilename):
-                        ts = self.list_fiveParts(square , 1, lhsChunkIdx, midChunkIdx, rhsChunkIdx)  # noqa
-                        tn = self.list_fiveParts(normal , 1, lhsChunkIdx, midChunkIdx, rhsChunkIdx)  # noqa
-                        summary = tS[0] + tS[1] + tS[3] + tS[2] + tS[4]
-                        curly =   tc[0] + tc[1] + tc[3] + tc[2] + tc[4]  # noqa
-                        square =  ts[0] + ts[1] + ts[3] + ts[2] + ts[4]  # noqa
-                        normal =  tn[0] + tn[1] + tn[3] + tn[2] + tn[4]  # noqa
-                        rhsChunkIdx -= 1
-                        midChunkIdx = summary[midChunkIdx:].index("S") + midChunkIdx
-                        continue
-
-                    # Try moving the chunk before.
-                    testcaseSuggestion.parts = p[0] + p[2] + p[1] + p[3] + p[4]
-                    if interesting(testcaseSuggestion):
-                        testcase = testcaseSuggestion
-                        log.info("<-Moving %s kept the file 'interesting'.", description)
-                        chunkLhsStart += chunkSize
-                        chunkLhsEnd += chunkSize
-                        chunkMidStart += chunkSize
-                        # pylint: disable=bad-whitespace
-                        tS = self.list_fiveParts(summary, 1, lhsChunkIdx, midChunkIdx, rhsChunkIdx)
-                        tc = self.list_fiveParts(curly  , 1, lhsChunkIdx, midChunkIdx, rhsChunkIdx)  # noqa
-                        ts = self.list_fiveParts(square , 1, lhsChunkIdx, midChunkIdx, rhsChunkIdx)  # noqa
-                        tn = self.list_fiveParts(normal , 1, lhsChunkIdx, midChunkIdx, rhsChunkIdx)  # noqa
-                        summary = tS[0] + tS[2] + tS[1] + tS[3] + tS[4]
-                        curly =   tc[0] + tc[2] + tc[1] + tc[3] + tc[4]  # noqa
-                        square =  ts[0] + ts[2] + ts[1] + ts[3] + ts[4]  # noqa
-                        normal =  tn[0] + tn[2] + tn[1] + tn[3] + tn[4]  # noqa
-                        lhsChunkIdx += 1
-                        midChunkIdx = self.list_nindex(summary, midChunkIdx, "S")
-                        stayOnSameChunk = True
-                        continue
-
-                    log.info("..Moving %s made the file 'uninteresting'.", description)
-                    chunkMidStart += chunkSize
-                    midChunkIdx = self.list_nindex(summary, midChunkIdx, "S")
+                # ###  MOVING-CHUNKS-SUBSTITUTE-START  ###
+                chunkStart += chunkSize
+                lhsChunkIdx = self.list_nindex(summary, lhsChunkIdx, "S")
+                continue
+                # ###  MOVING-CHUNKS-SUBSTITUTE-END  ###
 
-                lhsChunkIdx = origChunkIdx
-                if not stayOnSameChunk:
-                    chunkStart += chunkSize
-                    lhsChunkIdx = self.list_nindex(summary, lhsChunkIdx, "S")
+                # Moving chunks is still a bit experimental, and it can introduce reducing loops.
```

Comment:
Option 1 - `removing it`: undesirable because the concept will fade away in our minds over time too
Option 2 - `make it a separate strategy`: looks like more work is due here, but probably not suited for this PR
Option 3 - `add an option to enable it`: most reasonable, but I would like to push this out of the PR. Filed #54.

For now, I reverted the change that made this blob of code a large comment.